yarn-project/sequencer-client/src/publisher/sequencer-publisher.ts

```diff
@@ -264,6 +264,7 @@ export class SequencerPublisher {
    * @returns The slot and block number if it is possible to propose, undefined otherwise
    */
   public canProposeAtNextEthBlock(tipArchive: Buffer) {
+    // TODO: #14291 - should loop through multiple keys to check if any of them can propose
```

Comment:
This needs to be done in this PR doesn't it? Otherwise we will miss slots won't we?

Reply:
not in this version. All validator keys will be tied to the same proposer address (in this case the forwarder) so we only need to check once.
But with the update @LHerskind & @just-mitch are working on, then we'll no longer have attester, proposer, withdrawer, but just attester & withdrawer (iirc, pls correct me if I'm wrong) and won't be checking for `msg.sender` on L1